### DISCUSSION TITLE
Sole Treadmills

### DISCUSSION THREAD

**Discussion by jaantoineii** [cagnulein/qdomyos-zwift#2688] [Q&A]
I am new to github, so forgive me if I'm missing something. I'd like to know if this app will work with a 2021 Sole F85 treadmill. 

I'm planning to run Zwift from my PC and I want it to have the ability to control treadmill elevation, etc. The thing I like about this F85 is that it can run streaming apps like Netflix while in use. I have a 60" smart TV hooked up as a monitor for my PC. This way I could potentially Use Zwift on my television, but stream a show on the treadmill. Seems like this could be the best of both worlds.

Thank you in advance for your time!

**Comment by cagnulein** [comment]
Yes your treadmill is compatible! The fact if the treadmill has auto
inclination depends on the firmware. But the only way to discover it is to
try it!
Let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mar 22 ott 2024 alle 20:28 jaantoineii ***@***.***>
ha scritto:
